internal/config/local.go

```diff
@@ -59,7 +59,6 @@ func (l *Local) MeshInstance(result interface{}) error {
 		"installmode": "flat",
 		"installplatform": "kubernetes",
 		"installzone": " ",
-		"installversion": "0.7.0",
 		"mgmtaddr": "0.0.0.0:8000",
```

Comment:
what's the mgmtaddr?

Reply:
Management address I guess for portal ?

Reply:
Yep for the portal, although that's gonna go in the next commit.